TiddlyStow-ZH.html

```diff
@@ -1,335 +1,426 @@
-    }
-
-    function displayRecentFiles() {
-      const elem = document.getElementById("recent-files");
-      if (elem) {
-        twFileManager.getRecent().
-          then(recentFiles => {
-            if (recentFiles && recentFiles.length > 0) {
-              const list = document.createElement('ul');
-              for (const recent of recentFiles) {
-                const li = document.createElement('li'),
-                      button = Object.assign(document.createElement('button'), {
-                        type: 'button',
-                        onclick: openFile.bind(null, recent),
-                        innerText: recent.name
-                      });
-                li.appendChild(button);
-                list.appendChild(li);
-              }
-              const clearButton = Object.assign(document.createElement('button'), {
-                type: 'button',
-                onclick: clearRecent,
-                innerText: '清除最近的文件列表'
-              });
-              elem.innerText = '从最近使用过的文件开始：';
-              elem.appendChild(list);
-              elem.appendChild(clearButton);
-            }
-          })
-      }
-    }
-    window.onload = () => {
-      displayBrowserSupportMessage();
-      displayRecentFiles();
-    }
-  </script>
-</head>
-<body>
-    <div>
-      <h1>TiddlyStow</h1>
-      <p>这个简单的页面用于加载本地<a href="https://tiddlywiki.com">TiddlyWiki太微</a>程序文件。并将其保存到相同的本地文件中。</p>
-      <ul>
-        <li>在这里打开任何文件之前，请确保有一个备份!</li>
-        <li>一定要经常备份你的文件!</li>
-        <li>
-          并不是所有的浏览器都 <a href="https://caniuse.com/native-filesystem-api">支持本地文件特性</a>.
-          <span id="support-message"></span>
-        </li>
-        <li>仅适用于单个TiddlyWiki文件实例。</li>
-        <li>不喜欢在浏览器存储中保存最近的文件列表？点击 <a href="#norecent">这里</a> 并刷新页面。</li>
-        <li><a href="https://github.com/slaymaker1907/TW5-browser-nativesaver">TW5 browser nativesaver</a> 是一个类似的项目，但有更多的功能，并捆绑为TiddlyWiki插件。</li>
-        <li>源代码可在 <a href="https://github.com/btheado/tiddlystow">Github</a> 或者在浏览器中使用“查看页面源代码”。</li>
-        <li>加载Tiddlywiki文件后，刷新页面返回到文件选择器。</li>
-      </ul>
-      </p>
-      <br>
-      <button type="button" onclick="openFile()">从本地打开现有的文件...</button>
-      
-      <div id="recent-files"></div><br>
-      
-      
-      
-      <div>
-      从远程url打开新的wiki:
-      <ul>
-        <li><button type="button" onclick="openFromUrl('https://tiddlywiki.com/empty.html')">tiddlywiki.com/empty.html</button></li>
-        <li><button type="button" onclick="openFromUrl('https://tiddlywiki.com/prerelease')">tiddlywiki.com/prerelease</button></li>
-      </ul>
-      </div>
-      
-      
-      
-    </div>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="zh-CN">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>TiddlyStow - 使用浏览器文件系统api本地存储TiddlyWiki文件</title>
+  <link rel="shortcut icon" type="images/x-icon" href="https://zacharia2.github.io/TWPUB-BOOK/favicon.ico">
```

Comment:
Any reason why this is an absolute url? If you leave it as a relative url then it will be more flexible?

Reply:
Okay, it has been modified